ts/features/payments/details/store/actions/index.ts

```diff
@@ -35,7 +39,13 @@ export const paymentsTogglePagoPaCapabilityAction = createAsyncAction(
   "PAYMENTS_TOGGLE_PAGOPA_CAPABILITY_CANCEL"
 )<TogglePagoPaCapabilityPayload, void, NetworkError, void>();
 
+/** This action is used to close/show the PayPal banner into a payment method details of type paypal */
+export const paymentsPayPalBannerSetVisibleAction = createStandardAction(
```

Comment:
```suggestion
export const paymentsPayPalBannerSetIsClosedAction = createStandardAction(
```
I think that this name fit better the redux state property . What do you think?

Reply:
You're right, now it is more clear!

Addressed into c39f98d4ad0739e91dc31ce3c618d8333c519534